src/main/features/claim/views/confirmation.njk

```diff
@@ -20,52 +20,34 @@
       </div>
 
       <h2 class="heading-medium">{{ t('What happens next') }}</h2>
-      <p>{{ t('%s needs to respond to your claim by %s',
+      <p>{{ t('%s has until 4pm on %s to respond to your claim. They can ask for an extra 14 days if they need it.
```

Comment:
can you update to use interpolation instead of sprintf? here's an example from another page: 
```
<p>{{ t('The claimant can apply for a <a target="_blank" href="{{ link }}">County Court Judgment (CCJ)</a> against you if you don’t respond before the deadline.', { link: ['https://www.gov.uk/county-court-judgments-ccj-for-debt'] }) | safe }}</p>

Reply:
done